docs/tutorials/prowler-app-s3-integration.md

```diff
@@ -0,0 +1,334 @@
+                }
+            },
+            "Action": [
+                "s3:DeleteObject"
+            ],
+            "Resource": [
+                "arn:aws:s3:::<BUCKET NAME>/*ProwlerProBeacon"
+            ],
+            "Effect": "Allow"
+        }
+    ]
+}
+```
+
+#### S3 Put Object Permission
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Condition": {
+                "StringEquals": {
+                    "s3:ResourceAccount": "<BUCKET AWS ACCOUNT NUMBER>"
+                }
+            },
+            "Action": [
+                "s3:PutObject"
+            ],
+            "Resource": [
+                "arn:aws:s3:::<BUCKET NAME>/*"
+            ],
+            "Effect": "Allow"
+        }
+    ]
+}
+```
+
+#### S3 Get Object Permission
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Condition": {
+                "StringEquals": {
+                    "s3:ResourceAccount": "<BUCKET AWS ACCOUNT NUMBER>"
+                }
+            },
+            "Action": [
+                "s3:GetBucketLocation"
+            ],
+            "Resource": [
+                "arn:aws:s3:::<BUCKET NAME>"
+            ],
+            "Effect": "Allow"
+        }
+    ]
+}
+```
+
+???+ note
+    Replace `<BUCKET AWS ACCOUNT NUMBER>` with the AWS account ID that owns the destination S3 bucket, and `<BUCKET NAME>` with the actual bucket name.
+
+## Step 1: Navigate to Integrations
+
+To access S3 Integration:
+
+1. Navigate to  "Integrations"
+
+    ![Navigate to integrations](../tutorials/img/s3-integration-ui-1.png)
+
+2. The integrations page displays the available integrations
+
+## Step 2: Access S3 Integration Management
+
+1. Locate the Amazon S3 Integration card on the integrations page
+
+2. Click on the "Configure" button on the S3 integration card to access the dedicated management interface
+
+    ![Access S3 integration](../tutorials/img/s3-integration-ui-2.png)
+
+3. Review the S3 integration features and capabilities displayed on the page
+
+## Step 3: Add New S3 Integration
+
+### Configuration Step
+
+1. Click the "Add Integration" button to start the configuration process
+
+    ![Add integration button](../tutorials/img/s3-integration-ui-3.png)
+
+2. Complete the configuration form with the following details:
+
+    - **Cloud Providers:** Select the providers whose scan results should be exported to this S3 bucket
+    - **Bucket Name:** Enter the name of your target S3 bucket (e.g., `my-security-findings-bucket`)
+    - **Output Directory:** Specify the directory path within the bucket (e.g., `/prowler-findings/`)
+
+    ![Configuration form](../tutorials/img/s3-integration-ui-4.png)
+
+3. Click "Next" to proceed to the credentials step
+
+### Credentials Step
+
+1. Configure AWS authentication using one of the supported methods:
+
+    - **AWS SDK Default:** Use default AWS credentials from the environment
```

Comment:
Add a note here explaining that in Prowler Cloud this is the recommended credentials as the service has AWS credentials to assume an IAM role with an ARN matching 
```
"arn:aws:iam::*:role/Prowler*",
"arn:aws:iam::*:role/prowler*"
```

Reply:
Done.